ttnn/cpp/ttnn/operations/moreh/moreh_linear_backward/device/moreh_linear_backward_single_core_program_factory.cpp

```diff
@@ -0,0 +1,159 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <vector>
+
+#include "moreh_linear_backward_device_operation.hpp"
+#include "tt_dnn/op_library/moreh_helper_functions.hpp"
+#include "tt_metal/detail/util.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear_backward {
+
+MorehBiasAddBackwardOperation::SingleCoreProgramFactory::cached_program_t
+MorehBiasAddBackwardOperation::SingleCoreProgramFactory::create(
+    const operation_attributes_t& operation_attributes,
+    const tensor_args_t& tensor_args,
+    tensor_return_value_t& output_tensor) {
```

Comment:
Suggest change:
```C
tensor_return_value_t& bias_grad
```

Reply:
resolved in [bddbd72](https://github.com/tenstorrent/tt-metal/pull/12812/commits/bddbd720a89dd06cbd523315ac719a09b6987db1)